airflow/gcp/example_dags/example_automl_nl_text_classification.py

```diff
@@ -0,0 +1,108 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that uses Google AutoML services.
+"""
+import os
+
+import airflow
+from airflow import models
+from airflow.gcp.hooks.automl import CloudAutoMLHook
+from airflow.gcp.operators.automl import (
+    AutoMLTrainModelOperator,
+    AutoMLCreateDatasetOperator,
+    AutoMLImportDataOperator,
+    AutoMLDeleteDatasetOperator,
+    AutoMLDeleteModelOperator,
+)
+
+GCP_PROJECT_ID = os.environ.get("GCP_PROJECT_ID", "your-project-id")
+# For now only this location is supported
+GCP_AUTOML_LOCATION = os.environ.get("GCP_AUTOML_LOCATION", "us-central1")
+GCP_AUTOML_TEXT_CLS_BUCKET = os.environ.get(
+    "GCP_AUTOML_TEXT_CLS_BUCKET", "gs://"  # TODO override
```

Comment:
```suggestion
    "GCP_AUTOML_TEXT_CLS_BUCKET", "gs://"  # TODO override
```
Can you delete this comment? This can activate various tools used by developers.